components/brave_wallet_ui/page/screens/onboarding/components/recovery-phrase/recovery-phrase.tsx

```diff
@@ -0,0 +1,184 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+import * as React from 'react'
+
+// utils
+import { getLocale } from '$web-common/locale'
+import { unbiasedRandom } from '../../../../../utils/random-utils'
+
+// styles
+import {
+  RecoveryPhraseContainer,
+  RecoveryBubble,
+  FrostedGlass,
+  HiddenPhraseContainer,
+  EyeOffIcon,
+  RecoveryBubbleBadge
+} from './recovery-phrase.style'
+
+export interface SelectedPhraseWord {
+  index: number
+  value: string
+}
+
+interface Props {
+  hidden: boolean
+  recoveryPhrase: string[]
+  onClickReveal?: () => void
+  verificationModeEnabled?: boolean
+  onVerifyUpdate?: (doesWordOrderMatch: boolean) => void
+  onSelectedWordListChange?: (words: SelectedPhraseWord[]) => void
+}
+
+const FAKE_PHRASE_WORDS = new Array(12).fill('Fake')
+
+const SELECTED_WORD_ORDINALS = {
```

Comment:
Let's randomize which 3 words are chosen so we don't encourage a bad pattern of just remembering the first, third, last words rather than properly backing it up

Reply:
https://github.com/brave/brave-browser/issues/24683